modules/gapi/test/common/gapi_operators_tests_inl.hpp

```diff
@@ -17,6 +17,11 @@ TEST_P(MathOperatorMatScalarTest, OperatorAccuracyTest )
     auto fun_gapi = op.g_api_function;
     auto fun_ocv = op.ocv_function ;
 
+    if (op.name == "rev_operator/")
+        in_mat1.setTo(1, in_mat1 == 0);                               // avoiding zeros in divide input data
+    if (op.name == "operator/")
```

Comment:
????

Reply:
These are actions to avoid division by zero.
If you're asking about Scalar - its components are increased by 1, but only if they are equal 0.

Reply:
AFAIR there's no problems in OpenCV with division by zero. Go test it

Reply:
Changed string reference -> enum reference